From c69969c3b17f0017e9e6c36b3c7c9f1dbaa60d45 Mon Sep 17 00:00:00 2001
From: Sanjay R Mehta <sanju.mehta@amd.com>
Date: Tue, 6 Mar 2018 16:57:55 +0530
Subject: [PATCH 1118/4131] Added raven audio Kconfig

Signed-off-by: Sanjay R Mehta <sanju.mehta@amd.com>
---
 sound/soc/amd/Kconfig | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/sound/soc/amd/Kconfig b/sound/soc/amd/Kconfig
index 78187eb..dcdec36 100644
--- a/sound/soc/amd/Kconfig
+++ b/sound/soc/amd/Kconfig
@@ -2,3 +2,7 @@ config SND_SOC_AMD_ACP
 	tristate "AMD Audio Coprocessor support"
 	help
 	 This option enables ACP DMA support on AMD platform.
+config SND_SOC_AMD_ACP3x
+	tristate "AMD Audio Coprocessor-v3.x support"
+	help
+	 This option enables ACP v3.x I2S support on AMD platform.
-- 
2.7.4

