From 749880b5122f2cad82731cba6b3845ca364dcb29 Mon Sep 17 00:00:00 2001
From: "Arsalan H. Awan" <Arsalan_Awan@mentor.com>
Date: Mon, 19 Aug 2019 17:35:19 +0500
Subject: [PATCH 3/3] Allow enable DRI without DRI drivers

Signed-off-by: Andrei Gherzan <andrei@gherzan.ro>
Signed-off-by: Fabio Berton <fabio.berton@ossystems.com.br>
Signed-off-by: Arsalan H. Awan <Arsalan_Awan@mentor.com>
---
 meson.build       | 2 +-
 meson_options.txt | 6 ++++++
 2 files changed, 7 insertions(+), 1 deletion(-)

diff --git a/meson.build b/meson.build
index bb6e3083c20..7f4cfffc64a 100644
--- a/meson.build
+++ b/meson.build
@@ -136,7 +136,7 @@ with_dri_r200 = dri_drivers.contains('r200')
 with_dri_nouveau = dri_drivers.contains('nouveau')
 with_dri_swrast = dri_drivers.contains('swrast')
 
-with_dri = dri_drivers.length() != 0 and dri_drivers != ['']
+with_dri = get_option('dri') or (_drivers.length() != 0 and _drivers != [''])
 
 gallium_drivers = get_option('gallium-drivers')
 if gallium_drivers.contains('auto')
diff --git a/meson_options.txt b/meson_options.txt
index c0ce8d8c8bf..29031134617 100644
--- a/meson_options.txt
+++ b/meson_options.txt
@@ -35,6 +35,12 @@ option(
   description : 'enable support for dri3'
 )
 option(
+  'dri',
+  type : 'boolean',
+  value : false,
+  description : 'enable support for dri'
+)
+option(
   'dri-drivers',
   type : 'array',
   value : ['auto'],
-- 
2.11.1

